react-task/src/components/ButtonComponent.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import "bootstrap/dist/css/bootstrap.min.css";
+
+const ButtonComponent = ({ secondary, innerText }) => {
+  const isSecondaryButtonClass = secondary
+    ? "btn btn-light w-25 border"
+    : "btn btn-primary w-25";
+  return (
+    <button type="submit" class={isSecondaryButtonClass}>
```

Comment:
Add another prop for the type of button, and default it to `button`

```suggestion
    <button type={type} class={isSecondaryButtonClass}>
```